spec/fixtures/files/datahashes/core/exhibition1.json

```diff
@@ -26,19 +26,19 @@
     "exhibitionPersonPersonUlan": "We go",
     "exhibitionPersonOrganizationLocal": "breakup",
     "exhibitionPersonOrganizationUlan": "Again",
```

Comment:
In the CollectionSpace record, I changed the order of these fields to match the order in which the mapper processes the authorities (which should be the same order (left to right) as the columns appear in the CSV template). This is one of the limitations of the mapper that I'm working on documenting.